Weather App/UI/SearchVC/SearchViewController.swift

```diff
@@ -9,21 +9,28 @@ import UIKit
 
 class SearchViewController: UIViewController {
 
+    let searchView = SearchView()
+    var searchViewModel : SearchViewModel?
+    
+    override func loadView() {
```

Comment:
In `loadView` method, only setting the view is done. Nothing else. So, only `view = searchView`. Rest is done in `viewDidLoad` or other lifecycle methods.

Reply:
I moved setting the delegate to viewDidLoad, left just `view = searchView`.